src/cow-react/modules/limitOrders/containers/RateInput/styled.ts

```diff
@@ -20,10 +20,11 @@ export const Header = styled.div`
 export const MarketPriceButton = styled.button`
   background-color: ${({ theme }) => theme.bg1};
   color: ${({ theme }) => theme.text1};
+  white-space: nowrap;
   border: none;
   cursor: pointer;
   border-radius: 16px;
-  padding: 5px 10px;
+  padding: 5px 8px;
```

Comment:
I think we should also add styles when its disabled, something like `  &:disabled {
    opacity: 50%;
    cursor: auto;
    pointer-events: none;
  }`

Reply:
```
<styledEl.MarketPriceButton disabled={isLoadingExecutionRate || !executionRate}
```
Since it's a native button we have this out of the box